### PR TITLE
Changing IP DMA with nats-server

### PR DESCRIPTION
**Pull request by lustpieter**

### CODE REVIEW

**Review by LaurensVergote — APPROVED** [review]

File: user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ For a standalone DMA, i.e. a DMA that is not combined with other DMAs in a clust
 
    1. Restart the elasticsearch-service-x64 service.
 
-1. If the server has NATS installed:
+1. If the server has the NATS service installed:
```
Comment (EdithLansens) [inline user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md:46]:
```suggestion
1. If the server has NATS installed:
```

Can we leave this as "NATS" or does that change the content of the sentence, @lustpieter ?

File: user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md [hunk 2]
```diff
@@ -65,6 +65,14 @@ For a standalone DMA, i.e. a DMA that is not combined with other DMAs in a clust
 
       1. Press Enter twice, enter NATS, and press Y.
 
+1. If the server has the nats-server service installed:
```
Comment (EdithLansens) [inline user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md:68]:
```suggestion
1. If the server has nats-server installed:
```

Same question here, @lustpieter

File: user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md (hunk 1 above)
Comment (LaurensVergote) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md:46]: "the NATS service" is more complete, just saying NATS may be confusing

File: user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md (hunk 2 above)
Comment (LaurensVergote) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md:68]: same answer :)

File: user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md (hunk 1 above)
Comment (EdithLansens) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md:46]: Okay, thanks! :)

**Review by EdithLansens — APPROVED** [review]

**Pull request merged**